### PR TITLE
Homework5: Nikolai Potovin

### PR DESCRIPTION
**Pull request by adnike**
I will finish the second part for review soon

### CODE REVIEW

File: src/runner.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Модуль, использующий функции из pytasks.py
+def runner(*functions):
+    """Функция импорта из pytasks.py"""
+    if not functions:
+        from pytasks import count_characters
```
Comment (belovmd) [inline src/runner.py]: it's bad practices to import inside function
more of that we can collect all functions in module (maybe we'll add some tasks later)

File: src/runner.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        from pytasks import is_palindrome
+        print(generate_numbers())
+        print(count_characters())
+        print(fizzbuzz())
+        print(is_palindrome())
+    else:
+        import pytasks
+        for function_name in functions:
+            if hasattr(pytasks, function_name):
+                print(getattr(pytasks, function_name)())
```
Comment (belovmd) [inline src/runner.py:30]: we can add check if it's callable

**Review by belovmd — COMMENTED** [review]
1. don't delete init file
2. please name PR according our guidelines
3. please address these minor issues and add second task

**Comment by adnike** [comment]
Sent second part of HW5 with result files and fixes from first part. Hope that I defeated github this time.

File: src/imdb.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                "title": ' '.join(movie[3:-1]),
+                "rate": float(movie[2]),
+                "year": int(movie[-1][1:5]),
+            })
+except IOError:
+    print("File not found")
+
+try:
+    with open(os.path.join(
+            'c:' + os.sep, 'Homework5', 'top250_movies.txt'), 'w') as file:
```
Comment (belovmd) [inline src/imdb.py:33]: os.path.abspath

File: src/imdb.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+"""
+
+from collections import OrderedDict
+import os
+
+
+try:
+    film_list = []
+    with open(os.path.join(
+            'c:' + os.sep, 'Homework5', 'ratings.list'), 'r') as file:
```
Comment (belovmd) [inline src/imdb.py:15]: os.path.abspath

File: src/imdb.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        for i in range(28):
+            file.readline()
```
Comment (belovmd) [inline src/imdb.py:17]: file is a very bad name

File: src/imdb.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    print("File not found")
+
+dict_ratings = {}
+for film in film_list:
+    if film["rate"] in dict_ratings:
+        dict_ratings[film["rate"]] += 1
+    else:
+        dict_ratings[film["rate"]] = 1
+
+sortedDict = OrderedDict(sorted(dict_ratings.items()))
```
Comment (belovmd) [inline src/imdb.py:69]: camel case not python naming

**Review by belovmd — COMMENTED** [review]
8,5

**Pull request closed without merging**